modules/default/weather/weather.js

```diff
@@ -86,6 +86,9 @@ Module.register("weather", {
 			Log.warn("Your are using the deprecated config values 'useBeaufort'. Please switch to windUnits!");
 			this.windUnits = "beaufort";
 		}
+		if (this.config.showHumidity === true || this.config.showHumidity === false) {
```

Comment:
would be good if you then set the config showHumidity value to actually use wind so that we dont have to do the boolean check later in the code

Reply:
Done, and rebased to fix the conflict.